app/errors.py

```diff
@@ -0,0 +1,16 @@
+class VaccineError(Exception):
```

Comment:
```suggestion
class BaseVaccineError(Exception):
```
```suggestion
class AbstractVaccineError(Exception):
```

Reply:
but what's the difference?

Reply:
understanding of abstraction is better with base error, or abstract, I guess we'll refactor this later, since it failing the tests )